docs/src/pages/components/tabs/tabs.md

```diff
@@ -101,6 +101,10 @@ Here is an example of customizing the component. You can learn more about this i
 
 🎨 If you are looking for inspiration, you can check [MUI Treasury's customization examples](https://mui-treasury.com/styles/tabs/).
 
+## RTL tabs
+
+{{"demo": "pages/components/tabs/RTLTabs.js", "bg": true}}
+
```

Comment:
The RTL toggle in the docs already serves this purpose, also RTL is 2% of the usage, so It doesn't really matter

```suggestion
```

Also, the demo doesn't look accessible.